### DISCUSSION TITLE
How to do date range filter in table filters

### DISCUSSION THREAD

**Discussion by Rhythm-chicmicstudios** [primefaces/community#358] [PrimeNG] [answered]
I am using something like this

`<p-columnFilter type="date" [field]="col.field" display="menu" [showOperator]="false" [showAddButton]="false"></p-columnFilter>`
but I am not able to select the range as we could select in material angular date-picker

primeNg version : 13.4.5

**Comment by cetincakiroglu** [comment]
Hi,

Unfortunately, columnFilter does not support date range. You may use `contains` or `equals` to filter by date but it doesn't support date range.

**Reply by Rhythm-chicmicstudios** [reply, marked as answer]
Thanks For answering , 
Solved it by using p-calendar  inside columnFilter @cetincakiroglu
